src/main/scala/com/stackstate/pac4j/AkkaHttpWebContext.scala

```diff
@@ -45,13 +51,16 @@ case class AkkaHttpWebContext(request: HttpRequest) extends WebContext {
   }
 
   override def setResponseContentType(contentType: String): Unit = {
-    changes = changes.copy(contentType = contentType)
+    ContentType.parse(contentType) match {
```

Comment:
I like this parsing of the contentType in the setter already, lets do this for cookies aswell. We can make this into a separate MR (or put it on top of this one)